stac_api/infrastructure/config.py

```diff
@@ -13,21 +13,26 @@ class vedaSTACSettings(BaseSettings):
     memory: int = 8000  # Mb
 
     # Secret database credentials
-    pgstac_secret_arn: Optional[str] = Field(
+    stac_pgstac_secret_arn: Optional[str] = Field(
         None,
         description="Name or ARN of the AWS Secret containing database connection parameters",
     )
 
-    path_prefix: Optional[str] = Field(
+    stac_path_prefix: Optional[str] = Field(
         "",
         description="Optional path prefix to add to all api endpoints",
     )
 
+    domain_hosted_zone_name: Optional[str] = Field(
+        None,
+        description="Domain name for the cloudfront distribution",
+    )
+
     class Config:
         """model config"""
 
         env_file = ".env"
-        env_prefix = "VEDA_STAC_"
+        env_prefix = "VEDA_"
```

Comment:
I think this is a breaking change because it causes existing implementations to update the secrets stored in the secrets manager? I think we should address changing `<SCOPE>_VARNAME` across all constructs if we want to break the pattern (especially because it could be useful to share a variable that should be the same in the scopes of different constructs).

Is the change needed now, though, or can we keep scoping our environment variables?

Reply:
looks like staging has no `VEDA_STAC_` vars in the secrets store so this comment can be downgraded to a nit